src/fides/api/ops/api/v1/endpoints/connection_type_endpoints.py

```diff
@@ -71,14 +71,14 @@ def is_match(elem: str) -> bool:
             ]
         )
     if system_type == SystemType.saas or system_type is None:
+        registry: ConnectorRegistry = load_registry(registry_file)
         saas_types: List[str] = sorted(
             [
-                saas_type.value
-                for saas_type in SaaSType
-                if saas_type != SaaSType.custom and is_match(saas_type.value)
+                saas_type
+                for saas_type in registry.connector_types()
+                if saas_type != "custom" and is_match(saas_type)
```

Comment:
Now that we're moving away from the SaaSType enum we can stop using references to `custom`. It was originally meant to identify connector types that weren't in the enum list but now that any identifier is valid there is no need for this convention.

Reply:
good call! forgot about this consequence of the change.